docs/sphinx/source/conf.py

```diff
@@ -198,7 +199,9 @@
 #html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+html_sidebars = {
+    "**": ["search-field", "sidebar-nav-bs"]  # "sidebar-ethical-ads"
```

Comment:
default: `"**": ["search-field", "sidebar-nav-bs", "sidebar-ethical-ads"]`

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/sections.html#the-left-sidebar